go.mod

```diff
@@ -1,30 +1,26 @@
 module github.com/G-Research/fasttrack
 
-go 1.18
-
-replace (
-	github.com/mattn/go-sqlite3 v1.14.16 => github.com/jgiannuzzi/go-sqlite3 v1.14.17-0.20230327164124-765f25ea5431
-	gorm.io/driver/sqlite v1.4.3 => github.com/jgiannuzzi/gorm-sqlite v1.4.4-0.20221215225833-42389ad31305
-)
```

Comment:
Those are important in order to have SQLite3MultipleCiphers support!

Reply:
ook